modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -25,25 +25,38 @@ To integrate the built-in {ocp} OAuth server as an identity provider for {produc
 For security, Red Hat recommends first setting the *Minimum access role* to *None* while you complete setup. Later, you can return to the *Access Control* page to set up more tailored access rules based on user metadata from your identity provider.
 ====
 
-. To add access rules for users and groups accessing {product-title-short}, click *Add new rule* in the *Rules* section. For example:
-.. To give the *Admin* role to a user called `administrator`, you can use the following key-value pairs to create access rules:
-+
+. Optional: To add access rules for users and groups accessing {product-title-short}, click *Add new rule* in the *Rules* section, then enter the rule information and click *Save*. You will need attributes for the user so that you can configure their access. To get user information in {ocp}, you can use one of the following methods:
+* Use the link:https://docs.openshift.com/container-platform/4.11/rest_api/user_and_group_apis/user-user-openshift-io-v1.html[{ocp} API].
+* Click *User Management* -> *Users* -> *<username*> -> YAML.
+* Access the `k8s/cluster/user.openshift.io\~v1~User/<username>/yaml` file and note the values for `name`, `uid` (`userid` in {product-title-short}), and `groups`. 
+
+.Example
```

Comment:
Question that might just be me not following, is this `Example` for the whole procedure for "Configuring OpenShift Container OAuth server as an identity provider". To me, it reads like its own thing, but I wanted to ask.

Reply:
I'm struggling with this - it's just an example for how to add a rule (so step 7) - but if I try to format it below that as a sub-section, it is so many levels of bullets & text that it's unwieldy. Do you have any suggestions? What if I titled it "Rule example" or something? Open to any help :)

Reply:
screenshot of screen (add rules section at bottom): https://drive.google.com/file/d/1SgJca1Xmz_A68errZd2Ghk9XD9x-k004/view?usp=sharing

Reply:
I gave up on the table - does the text-based approach look ok? I think it's less confusing - what I didn't want was for a user to think "I have to configure everything in that table" because that would be overkill.

Reply:
I think it's fine.